hooks/useCreateOrder.ts

```diff
@@ -16,14 +16,14 @@ export const useCreateOrder = () => {
     const { maker, isCollectionApproved, isTransferManagerApproved } =
       await hypercertExchangeClient.createFractionalSaleMakerAsk({
         startTime: Math.floor(Date.now() / 1000), // Use it to create an order that will be valid in the future (Optional, Default to now)
-        endTime: Math.floor(Date.now() / 1000) + 365 * 24 * 60 * 60, // If you use a timestamp in ms, the function will revert
-        price: parseEther("1"), // Be careful to use a price in wei, this example is for 1 ETH
+        endTime: Math.floor(Date.now() / 1000) + (100 * 365 * 24 * 60 * 60), // If you use a timestamp in ms, the function will revert
```

Comment:
Why 100 years?

Reply:
Just to maintain consistency with the previous 17 claims that were minted with 100 years